friends/src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import { BrowserRouter as Router, withRouter } from 'react-router-dom';
+
+const AppWithRouter = withRouter(App)
```

Comment:
Great job using with router to set this up and get it working to where it is passing in the history so you can use that for routing in app.js